### ISSUE TITLE
data

### ISSUE THREAD

**Issue by liuxiaocong20** [bhagya-hettige/MedGraph#1]
Hi! Great work! Is the data public available?

**Comment by bhagya-hettige** [comment]
No. Unfortunately, the data we used is from a proprietary dataset from Monash Health Hospital in Australia. However, there are some other public EMR datasets: MIMIC-III and CMS etc. I can provide you the links if interested.
